hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/discovery/HazelcastCloudDiscovery.java

```diff
@@ -156,4 +192,37 @@ public static String createUrlEndpoint(String cloudBaseUrl, String cloudToken) {
         return cloudBaseUrl + CLOUD_URL_PATH + cloudToken;
     }
 
+    /**
+     * Contains the response of the discovery request to the coordinator endpoint.
+     */
+    static final class DiscoveryResponse {
+        private final Map<Address, Address> privateToPublicAddresses;
+        private final List<Address> privateMemberAddresses;
+
+        DiscoveryResponse(Map<Address, Address> privateToPublicAddresses, List<Address> privateMemberAddresses) {
+            this.privateToPublicAddresses = privateToPublicAddresses;
+            this.privateMemberAddresses = privateMemberAddresses;
+        }
+
+        /**
+         * Returns the list of private member addresses.
+         * <p>
+         * In TPC enabled clusters, this list only contains the private
+         * addresses of the classic Hazelcast ports.
```

Comment:
Did you mean TPC disabled?

Reply:
No, it was just to point out that even in TPC-enabled clusters, this list just contains the classic ports. Let me clarify it a bit more

Reply:
Ok got it. Maybe we can say this list does not contain tpc port addresses.

Reply:
https://github.com/hazelcast/hazelcast/pull/24378/commits/254ac37b5d003322c57e47ce8d5ac6d0ad7fe7be